contracts/Loans.vy

```diff
@@ -301,7 +304,10 @@ def _loanPayableAmount(
     _timePassed: uint256,
     _interestAccrualPeriod: uint256
 ) -> uint256:
-    return (_amount - _paidAmount) * (10000 * _maxLoanDuration + _interest * (_timePassed + _interestAccrualPeriod)) / (10000 * _maxLoanDuration)
+    minimum_interest_period: uint256 = DURATION_30_DAYS
+    if _maxLoanDuration <= DURATION_89_DAYS:
```

Comment:
The `_maxLoanDuration` should be compared against `DURATION_30_DAYS`, right?

The rules are:
* The loan duration is less or equal than 7d -> minimum interest to pay is of 7 days
* The loan duration is between 8d and 30d -> minimum interest to pay is of 7 days
* The loan duration is between 31d and 90d -> minimum interest to pay is of 30 days

Reply:
Ok, got it in a different way: the protocol would have always the durations for 7,30 or 90 days (with adjusted interest) even if the selected period was different (eg a 60 days loan would be 90 days in the contract). On a second thought, I guess that would mess with the defaults.
Anyway, changing it.